scripts/create_dummy_profiles_json.py

```diff
@@ -0,0 +1,392 @@
+
+Example run of script:
+python scripts/create_dummy_profiles_json.py 5 -e haavikko@dataflow.fi -p city_profiles.json -c customer_profiles.json
+
+This script creates two JSON data files:
+
+1. JSON data file that is suitable for importing to the Helsinki profile service,
+   specified with the -p command line argument.
+   For more details, see open-city-profile project, models.Profile.import_customer_data
+
+2. JSON data file that contains the customer profiles, specified with the -c command line argument.
+   See CustomerProfileManager.import_customer_data for details.
+
+Use -e argument to specify the email addresses for dummy customers.
+If multiple emails are specified, then each dummy customer is assigned a random address from the list.
+Use -e RANDOM to generate random emails.
+
+NOTE: the second file needs additional processing before importing.
+CustomerProfile needs to have UUID of the profile in the Helsinki profile service,
+and the UUID is generated by the Helsinki profile service at import time.
+When file (1) is imported, the Helsinki profile admin UI view returns a JSON file containing
+a mapping customer_id -> actual UUID, by default named export.json
+
+After processing, run merge_profile_uuids.py script as follows:
+python scripts/merge_profile_uuids.py -b <profiles_file_name> -m export.json
+This script creates files named berth_profiles_<NN>.json, which can be imported via the admin UI.
+
+api_url - use to fetch data about berths, piers and harbors. Running this script only reads data from the API,
+it does not change data in the server. The data is used to:
+* assign dummy customers to berths that are actually available
+* ensure dummy boats that fit in their berth
+
+-e flag can be used to specify the email address(es) to use for dummy users.
+If we actually send lots of emails while testing, then the
+recipient address should probably be a real email address, not something random.
+Mailgun (email service used as of 2020-12) has spammer detection logic and if it sees a lot of
+undeliverable mail, it might decide we are up to no good. "Mailgun has thresholds in place that if
+exceeded will result in a domain being temporarily disabled."
+
+For more help on available arguments, run the script with -h flag.
+
+Last name of the generated profiles starts with "Berth" so that the generated profiles
+can be identified in the database.
+
+"""
+
+import argparse
+import datetime
+import json
+import os
+import random
+import string
+import sys
+import traceback
+import uuid
+
+import django
+from faker import Faker
+from gql import Client, gql
+from gql.transport.requests import RequestsHTTPTransport
+
+fake = Faker("fi_FI")
+
+HELSINKI_PROFILE_TEMPLATE = {
+    "customer_id": lambda: str(uuid.uuid4()),
+    "first_name": fake.first_name,
+    "last_name": lambda: "Berth" + fake.last_name(),
+    "email": lambda: fake.pystr(min_chars=10, max_chars=10)
+    + "@"
+    + fake.pystr(min_chars=10, max_chars=10)
+    + ".com",
```

Comment:
I think this could make it a bit more readable :) The line breaks with the `+` first made me think they wre different entries 😅 
```suggestion
    "email": lamda: f"{fake.pystr(min_chars=10, max_chars=10)}@{fake.pystr(min_chars=10, max_chars=10)}.com"
```

Reply:
black formatter wanted to format it that way... I'll change it into a function to make it look better